rhino/src/main/java/org/mozilla/javascript/AbstractEcmaObjectOperations.java

```diff
@@ -232,4 +242,65 @@ static void put(Context cx, Scriptable o, int p, Object v, boolean isThrow) {
             base.put(p, o, v);
         }
     }
+
+    /**
+     * Implement the ECMAScript abstract operation "GroupBy" defined in section 7.3.35 of ECMA262.
+     *
+     * @param cx
+     * @param scope
+     * @param items
+     * @param callback
+     * @param keyCoercion
+     * @see <a href="https://tc39.es/ecma262/#sec-groupby"></a>
```

Comment:
Sorry, just seeing this now: the link to the spec doesn't reference a specific version of the spec, which it should 😐